livepeer/go-livepeer
# common: Get gpus using pci info if gpu.GrahpicsCards is empty

**What does this pull request do? Explain your changes. (required)**
Fix `-nvidia all` flag not detecting GPUs.
On VMs `ghw.GPU()` does return an empty array. We try to fallback to `ghw.PCI()` by picking only graphics cards. `pci.Device.Name`, `pci.Driver` and `pci.Device.Class.Name` are queried to detect display controller PCI devices

**Specific updates (required)**
- Check `pci.Device.Name`, `pci.Driver` and `pci.Device.Class.Name` if `gpu.GraphicsCards` is empty

**How did you test each of these updates (required)**
- Launch a GCP VM instance that has an Nvidia GPU
- run `./livepeer -orchestrator -transcoder -network offchain -nvidia all`
- Observe that GPUs are identified


**Does this pull request close any open issues?**
Fixes #2339


**Checklist:**
